Dockerfile

```diff
@@ -44,10 +44,10 @@ COPY --chown=app:app spec/internal_test_hyku/Gemfile ./spec/internal_test_hyku/G
 COPY --chown=app:app spec/internal_test_hyku/Gemfile.lock ./spec/internal_test_hyku/Gemfile.lock
 
 ENV CFLAGS=-Wno-error=format-overflow
-RUN bundle config build.nokogiri --use-system-libraries \
-	&& bundle config set without 'production' \
-	&& bundle config set with 'aws development test postgres' \
-	&& setuser app bundle install --jobs=4 --retry=3
+RUN bundle config build.nokogiri --use-system-libraries && \
+    bundle config set without "production" && \
+    bundle config set with "aws development test postgres" && \
+    setuser app bundle install --jobs=4 --retry=3
 
 RUN chmod 777 .bundle/config # Otherwise `app` owns this file and the host cannot run bundler commands
```

Comment:
As we saw with my setup during hyrax_autopopulation  gem, we still could not run bundler outside docker with doing another **chmod 777** in the host mchine, do you want to try this instead 

      RUN chmod 777  -R .bundle/*